Lib/test/test_clinic.py

```diff
@@ -2250,7 +2265,7 @@ class Foo "" ""
                 self.expect_failure(block, expected_error, lineno=3)
 
     def test_getset_no_class(self):
-        for annotation in "@getter", "@setter":
+        for annotation in ("@getter", "@setter"):
```

Comment:
@erlend-aasland This was introduced from https://github.com/python/cpython/pull/113278, but I felt a little uncomfortable with an implicit declaration. If you want to keep this as-is, please let me know, and I will revert it. 
cc @AlexWaygood

Reply:
There is no need to change this; loops with tuples come in both variants in `test_clinic.py`. The added parentheses has no effect.

```suggestion
        for annotation in "@getter", "@setter":
```

Reply:
I also prefer it without the parentheses here

Reply:
Done!

Reply:
Thanks!